onnxruntime/core/graph/model.cc

```diff
@@ -139,11 +139,18 @@ Model::Model(std::unique_ptr<ModelProto> model_proto, const IOnnxRuntimeOpSchema
 
   auto domain_map = schema_registry->GetLatestOpsetVersions(false);
   for (const auto& domain : domain_map) {
-    if (domain_to_version.find(domain.first) == domain_to_version.end()) {
+    auto iter = domain_to_version.find(domain.first);
```

Comment:
The major differ between this change and #2413 is: here it retrieves the domain version mapping from a local schema_registry, which has better support for custom ops and custom registries.